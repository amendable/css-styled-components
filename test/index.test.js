import React from 'react';
import renderer from 'react-test-renderer';
import cssResolver from '../src';

const resolver = cssResolver({});

test('returns correct component', () => {
  const resolved = resolver.resolve({ key: 'css', value: { backgroundColor: 'red' }, props: { component: 'div' } });
  const Component = resolved.component;
  const tree = renderer.create(<Component>Test</Component>).toJSON();

  expect(tree).toMatchSnapshot();
})
