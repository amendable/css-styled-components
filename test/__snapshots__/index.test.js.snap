// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct component 1`] = `
<div
  className="sc-bdVaJa gyiZyl"
>
  Test
</div>
`;
